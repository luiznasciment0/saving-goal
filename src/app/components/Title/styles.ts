import styled from 'styled-components'

export const Wrapper = styled.div`
  display: grid;
  place-items: center;
  background-color: #f4f8fa;
  height: 7rem;
`
