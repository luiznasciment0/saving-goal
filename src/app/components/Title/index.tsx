import { Wrapper } from './styles'
import SubtitleXS from 'app/components/SubtitleXS'

const Title = () => (
  <Wrapper>
    <SubtitleXS>
      Let&apos;s plan your <strong>saving goal.</strong>
    </SubtitleXS>
  </Wrapper>
)

export default Title
