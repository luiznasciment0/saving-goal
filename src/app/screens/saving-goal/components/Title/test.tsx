import { render, screen } from '@testing-library/react'

import Title from '.'

describe('<Title />', () => {
  it('should render the title', () => {
    const { container } = render(<Title />)

    expect(
      screen.getByRole('heading', { name: /Let's plan your saving goal./i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
