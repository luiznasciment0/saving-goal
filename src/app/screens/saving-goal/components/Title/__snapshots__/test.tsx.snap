// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render the title 1`] = `
.c0 {
  display: grid;
  place-items: center;
  background-color: #f4f8fa;
  height: 7rem;
}

.c1 {
  font-size: 1.8rem;
  line-height: 2.16rem;
  font-family: 'Work Sans';
  color: #1b31a8;
  font-weight: 400;
}

<div
  class="c0"
>
  <h1
    class="c1"
    color="primaryBlue"
  >
    Let's plan your 
    <strong>
      saving goal.
    </strong>
  </h1>
</div>
`
